github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -489,7 +489,7 @@ periodics:
     testgrid-dashboards: kubevirt-periodics
     testgrid-days-of-results: "60"
   cluster: prow-workloads
-  cron: 0 22, 11 * * *
+  cron: 0 23 * * 5,6
```

Comment:
Please consider spreading it across the week, so it will have a better chance of catching problems earlier.
AFAIK Less code is being pushed during the weekend so it will probably test the same code on Friday and Saturday.

Reply:
We don't want to run it when there are other jobs, it just take resources
It is a lane that is going to be deprecated, it is just step before.

Reply:
I reduced it to one time a week, there is no need for more, and then we can agree maybe easier about this discussion